packages/next-swc/crates/core/Cargo.toml

```diff
@@ -16,18 +16,18 @@ pathdiff = "0.2.0"
 serde = "1"
 serde_json = "1"
 styled_components = "0.17.0"
-swc = "0.138.0"
-swc_atoms = "0.2.7"
-swc_common = { version = "0.17.9", features = ["concurrent", "sourcemap"] }
-swc_css = "0.94.0"
-swc_ecma_loader = { version = "0.28.0", features = ["node", "lru"] }
-swc_ecmascript = { version = "0.123.0", features = ["codegen", "minifier", "optimization", "parser", "react", "transforms", "typescript", "utils", "visit"] }
+swc = "0.140.1"
+swc_atoms = "0.2.9"
+swc_common = { version = "0.17.10", features = ["concurrent", "sourcemap"] }
+swc_css = "0.98.0"
+swc_ecma_loader = { version = "0.29.0", features = ["node", "lru"] }
+swc_ecmascript = { version = "0.125.0", features = ["codegen", "minifier", "optimization", "parser", "react", "transforms", "typescript", "utils", "visit"] }
```

Comment:
Various swc crates have patch to reduce the binary size, but I'm not sure if it's applied in this version.

Reply:
Yeah, no idea. I'll defer to https://github.com/vercel/next.js/pull/35133 -- I'm trying to get a patch for parsing behavior that was associated with `@swc/core` 1.2.148, shipped 5 days ago and happy to toss this PR out in favor of yours.

Reply:
Thank you!
I fixed the rust version issue and updated https://github.com/vercel/next.js/pull/35133 to use the fixed version